Source/WebCore/Modules/pdfjs-extras/content-script.js

```diff
@@ -27,18 +27,27 @@ const PDFJSContentScript = {
         // Disable the findbar provided by PDF.js
         delete PDFViewerApplication.supportsIntegratedFind;
         PDFViewerApplication.supportsIntegratedFind = true;
+
+        // Hide sidebar on load
+        PDFViewerApplicationOptions.set("sidebarViewOnLoad", 0);
     },
 
     setPageMode({ pages, continuous }) {
         PDFViewerApplication.pdfViewer.spreadMode = pages == "two" ? 1 : 0;
 
-        if (continuous) {
+        if (continuous)
             PDFViewerApplication.pdfViewer.scrollMode = 0;
-            PDFViewerApplication.pdfViewer.currentScaleValue = "page-width";
-        } else {
+        else
             PDFViewerApplication.pdfViewer.scrollMode = 3;
+
+        this.autoResize();
+    },
+
+    autoResize() {
+        if (PDFViewerApplication.pdfViewer.spreadMode == 1) // Two pages
```

Comment:
Are there named constants for `spreadMode` and `scrollMode`? Would be nice to use one if possible.

Reply:
There are, but they're not exposed, unfortunately.

Reply:
Can we redefine (since we're already relying on the integer value) or get them exposed in PDF.js?